### PR TITLE
Milestone 2: Works and About me section

### PR DESCRIPTION
**Pull request by amejid**
## Personal portfolio milestone 2: Works and About me section
In this milestone I:
- Added the section to showcase projects
- Added about me section with tools

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -151,5 +151,293 @@ <h2 class="headline-job">I’m a software developer</h2>
...
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+      </div>
```
Comment (chrystalme) [inline index.html]: Kindly wrap your projects in a `<section>` tag. For semantic purposes and to make your page more SEO friendly.

File: index.html [hunk 2]
```diff
@@ -151,5 +151,293 @@ <h2 class="headline-job">I’m a software developer</h2>
...
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
```
Comment (chrystalme) [inline index.html]: Kindly use `flex` for the alignment of your content in this card and all similar cards. This is a project requirement.

**Review by chrystalme — CHANGES_REQUESTED** [review]
# Hi @amejid , 
giphy_close

You have done a good job so far! 👏🏽 👏🏽 
However, there are some issues that you still need to work on to prepare your project for further evaluation. 

## Highlights 💡
- Well configured Linters ✔️ 
- Following Gitflow guidelines ✔️ 
- Professional git commit statements ✔️ 
 
## Suggested changes ♻️ 
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -151,5 +151,293 @@ <h2 class="headline-job">I’m a software developer</h2>
...
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
```
Comment (Ismailco) [inline index.html]: - [ ] The project title should use ```hx``` tag, this is a requirement. please change the ```<p>``` to a ```<hx>``` tag

File: styles.css [hunk 4]
```diff
@@ -82,3 +82,181 @@ nav {
...
+  text-align: center;
+}
+
+.project-tools {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+.project-tools li {
```
Comment (Ismailco) [inline styles.css:144]: - [ ] You can decrease the padding/margin on the tools section so your design matches the Figma design 

Screenshot_20220526_185509

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @amejid,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @amejid,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by amejid** [comment]
Thank you for reviewing and approving my code.

File: index.html (hunk 1 above)
Comment (amejid) [reply index.html]: Thank you for reviewing my project, I will make the necessary corrections as suggested.

File: index.html (hunk 2 above)
Comment (amejid) [reply index.html]: Noted. I will work on that.

File: index.html (hunk 3 above)
Comment (amejid) [reply index.html]: Noted. I will work on it.

File: styles.css (hunk 4 above)
Comment (amejid) [reply styles.css:144]: Thank you for reviewing my project, I will make the necessary corrections as suggested.

**Pull request merged**